build/docker-compose.yml

```diff
@@ -34,3 +36,16 @@ services:
     depends_on:
      - backend
      - frontend
+  postgres:
+    container_name: bt_postgres
+    image: postgres:12.6
+    volumes:
+      - ./bt_main.dump:/bt_main.dump
+      - ./postgres-data:/var/lib/postgresql/data
+    ports:
+      - "5432:5432"
+    environment:
+      - "POSTGRES_DB=bt_main"
```

Comment:
I was wrong, based on the seed file it might actually need to be db `bt`.
```
curl https://storage.googleapis.com/berkeleytime/public/bt_seed.sql.gz | gzip -d | docker exec -i postgres psql --username bt --dbname bt_main
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0psql: error: FATAL:  database "bt_main" does not exist
  0 23.0M    0 49354    0     0  92596      0  0:04:21 --:--:--  0:04:21 92423
curl: (23) Failure writing output to destination
```

Reply:
Use `make postgres` to create the server & db with db = bt_main and user, password = bt